crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -610,6 +616,12 @@ impl SparseSets {
         }
     }
 
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably be pub(crate). There's no way for a user of bevy_ecs to get a &mut SparseSets` without being unsound.